### ISSUE TITLE
GetVarAction Timeout in Asterisk 13 and 15 (ami 2.10.0 and 4.0.0)

### ISSUE THREAD

**Issue by BuilovAlexey** [AsterNET/AsterNET#123]
Hello.

I successfully used this lib (but earlier version) with Asterisk 1.2, 1.6. But i installed fresh version of Asterisk.net and checked all operations with new Asterisks. 
When I installed "free pbx" base on Asterisk 13 (ami 2.10.0 version) and Asterisk 15(ami 4.0.0 version) - i can't use GetVarAction.

This code:
```
ManagerResponse responseSrc;
String valueSrc = " ";
try {
    GetVarAction getVarActioneSrc = new GetVarAction(e.Channel, "CALLERID(num)");
    responseSrc = Asterisk1cFriendshipFacade.Instance._connection.SendAction(getVarActioneSrc, 8000);
    if (responseSrc.IsSuccess())
        valueSrc = responseSrc.GetAttribute("Value");
} catch (Exception ex){
     valueSrc = e.CallerIdName;
} 
```
After executing command SendAction I got ex = {"Timeout waiting for response to GetVar"}" (System.Collections.ListListDictionaryInternal type)
 
I checked GetVarAction with "CALLERID(all)" - same result.

RedirectAction, HangupAction, OriginateAction - work fine.

What can it may?

**Comment by Deantwo** [comment]
Is this still an issue?

**Comment by BuilovAlexey** [comment]
I do not know, you tell me :) 
The bug was reproduced in November 2017. Did you fix it in new commits? Did you check it?

**Comment by skrusty** [comment]
@BuilovAlexey with respect, we're not here to do everything, this is an open source project. Your input on if an issue has been fixed or not would be greatly appreciated, but if you're unable or not willing to, then we will close the issue until it's either reported again, or something is willing to review it.

**Comment by Deantwo** [comment]
I am just going through a few old issues.

I don't quite understand this issue so I don't know if it was fixed or if you found a workaround.

**Comment by BuilovAlexey** [comment]
@skrusty No, I did not correct this bug in this project. I just went back to the earlier version of the project, where there was not this error.
You do not close issue if it is not solved by anyone? So let it be open I think. And other developers could see it and do not ask again

**Comment by skrusty** [comment]
@BuilovAlexey My point is you don't need to be rude to other contributors, simply saying you're unaware if the issue is fixed, or pointing to how you resolved it by using an older version would be useful.

**Comment by Deantwo** [comment]
@BuilovAlexey, which version did you downgrade to?

**Comment by Deantwo** [comment]
I can get this to timeout on the current version, but I haven't a clue about how it SHOULD work.

**Comment by Deantwo** [comment]
@BuilovAlexey, did you want to help solve this issue? At least tell us which version you downgraded to so we know when it broke?
